### ISSUE TITLE
System.js is unable to find dist/leaflet.js

### ISSUE THREAD

**Issue by shaunluttin** [benib/aurelia-leaflet#4]
Steps to reproduce in an existing Aurelia app:
1. `jspm install github:benib/aurelia-leaflet`
2. In main.ts add `.plugin('benib/aurelia-leaflet');`
3. `gulp watch`
4. Firefox > localhost:9000

Network Panel

![capture](https://cloud.githubusercontent.com/assets/2066637/15937778/246b1d86-2e25-11e6-94bf-0cb7f67cc144.PNG)

Web Console

![capture](https://cloud.githubusercontent.com/assets/2066637/15937802/3ce26e6e-2e25-11e6-99b7-50225ebe0d0c.PNG)

**Comment by shaunluttin** [comment]
The problem is `.\jspm_packages\github\benib\aurelia-leaflet@0.1.1\index.js`.

```
frameworkConfig.globalResources('leaflet');
```

Needs to be: 

```
frameworkConfig.globalResources('./leaflet');
```

**Comment by benib** [comment]
Thanks for the very detailed report. This is fixed in version `0.1.2`.
